dynaml/parser_test.go

```diff
@@ -135,16 +159,74 @@ var _ = Describe("parsing", func() {
 			parsesAs(
 				`"foo" - bar - merge`,
 				SubtractionExpr{
-					StringExpr{"foo"},
 					SubtractionExpr{
+						StringExpr{"foo"},
 						ReferenceExpr{[]string{"bar"}},
-						MergeExpr{},
 					},
+					MergeExpr{},
 				},
 			)
 		})
 	})
 
+	Describe("multiplication", func() {
+		It("parses nodes separated by *", func() {
+			parsesAs(
+				`"foo" * bar`,
+				MultiplicationExpr{
+					StringExpr{"foo"},
+					ReferenceExpr{[]string{"bar"}},
+				},
+			)
+
+			parsesAs(
+				`"foo" * bar * merge`,
+				MultiplicationExpr{
+					MultiplicationExpr{
+						StringExpr{"foo"},
+						ReferenceExpr{[]string{"bar"}},
+					},
+					MergeExpr{},
+				},
+			)
+		})
+	})
+	
+	Describe("division", func() {
+		It("parses nodes separated by *", func() {
```

Comment:
It appears that this `It` block's description is copied and pasted from above. In this case we believe that `*` should be replaced with `/`.